test/contracts.py

```diff
@@ -21,17 +20,69 @@ class BridgeContractConfig:
     next_swap_id: int
 
 DefaultBridgeContractConfig = BridgeContractConfig(
-    cap = "250000000000000000000000000",
-    reverse_aggregated_allowance = "3000000000000000000000000",
-    reverse_aggregated_allowance_approver_cap = "3000000000000000000000000",
-    lower_swap_limit = "1",
-    upper_swap_limit = "1000000000000000000000000",
-    swap_fee = "0",
-    paused_since_block = 18446744073709551615,
-    denom = "atestfet",
-    next_swap_id = 0
+    cap="250000000000000000000000000",
+    reverse_aggregated_allowance="3000000000000000000000000",
+    reverse_aggregated_allowance_approver_cap="3000000000000000000000000",
+    lower_swap_limit="1",
+    upper_swap_limit="1000000000000000000000000",
+    swap_fee="0",
+    paused_since_block=18446744073709551615,
+    denom="atestfet",
+    next_swap_id=0
 )
 
+
+@dataclass_json
+@dataclass
+class CW20BalanceObject:
+    amount: int
+    address: str
+
+
+@dataclass_json
+@dataclass
+class CW20ContractConfig:
+    name: str
+    symbol: str
+    decimals: int
+    initial_balances: list[dict[str, str]]
+
+
+DefaultCW20ContractConfig = CW20ContractConfig(
+    name="test coin",
+    symbol="TEST",
+    decimals=6,
+    initial_balances=[{
+            "amount": "3000000",
+            "address": "fetch1wurz7uwmvchhc8x0yztc7220hxs9jxdjdsrqmn"  # validator address
+    }]
+)
+
+
+class CW20Contract(LedgerContract):
+
+    def __init__(self, client: LedgerClient, admin: Wallet, cfg: CW20ContractConfig):
+        url = "https://github.com/CosmWasm/cw-plus/releases/download/v0.14.0/cw20_base.wasm"
+        if not os.path.exists(".contract"):
+            os.mkdir(".contract")
+        try:
+            temp = open(".contract/cw20.wasm", "rb")
+            temp.close()
+        except:
+            contract_request = requests.get(url)
+            file = open(".contract/cw20.wasm", "wb")
+            file.write(contract_request.content)
+            file.close()
```

Comment:
I think this can be simplified by using a context manager:

```suggestion
            with open(".contract/cw20.wasm", "wb") as file:
              file.write(contract_request.content)
```